src/main/java/org/mockito/ArgumentMatcher.java

```diff
@@ -125,4 +132,12 @@
      * @return true if this matcher accepts the given argument.
      */
     boolean matches(T argument);
+
+    /**
+     * Provide a meaningful description of the matcher in error messages.
+     *
+     * @return a description of the values that the matcher matches, e.g. the
+     * {@link ArgumentMatchers#anyString()} matcher returns {@code "<any string>"}.
+     */
+    String toString();
```

Comment:
So my Java experience is probably lackluster here, but is adding this method to the interface a breaking change? Given that you didn't make other changes and the tests still pass, I assume it isn't. Just something I don't know 😛